container/container_contract.go

```diff
@@ -93,21 +93,6 @@ func _deploy(data interface{}, isUpdate bool) {
 	if isUpdate {
 		args := data.([]interface{})
 		common.CheckVersion(args[len(args)-1].(int))
-		storage.Delete(ctx, common.LegacyOwnerKey)
-
-		// Migrate container estimation keys.
-		it := storage.Find(ctx, []byte(estimateKeyPrefix), storage.DeserializeValues)
```

Comment:
I am afraid this migration code should stay.

Considering we have [v0.14.1](https://github.com/nspcc-dev/neofs-contract/tree/v0.14.1) deployed in the network, this version does not include changes from #216